### PR TITLE
Remove user bots

### PR DESCRIPTION
**Pull request by bdistin**
**Please describe the changes this PR makes and why it should be merged:**
One of the milestone updates for v12 is the removal of user bot support (as it's been made clear use of which is against the TOS anyway). This pr attempts to accomplish that milestone.

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [x] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/ClientUser.js [hunk 1]
```diff
@@ -20,75 +15,8 @@ class ClientUser extends Structures.get('User') {
...
-     * @type {?boolean}
-     */
-    this.premium = typeof data.premium === 'boolean' ? data.premium : null;
-
-    /**
-     * If the user has MFA enabled on their account
-     * <warn>This is only filled when using a user account.</warn>
-     * @type {?boolean}
-     */
-    this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
```
Comment (SpaceEEC) [inline src/structures/ClientUser.js]: This field is actually filled when using a bot account.
`true` when the owner has mfa enabled, `false` if not.

File: src/client/Client.js [hunk 2]
```diff
@@ -96,17 +95,10 @@ class Client extends BaseClient {
      */
     this.channels = new ChannelStore(this);
 
-    /**
-     * Presences that have been received for the client user's friends, mapped by user IDs
-     * <warn>This is only filled when using a user account.</warn>
-     * @type {ClientPresenceStore<Snowflake, Presence>}
-     */
-    this.presences = new ClientPresenceStore(this);
```
Comment (devsnek) [inline src/client/Client.js]: this must remain for bots with friends and to handle the presence of the client

**Comment by ShayBox** [comment]
I use fetchApplication on the bot user itself to get info, maybe just remove the param so it can only get the logged in bot?

**Comment by bdistin** [comment]
That's what I did? 51da66ec37019f635c2634e7fe3f0838e988ec00

**Comment by Zipdox** [comment]
Looks like I will be stuck with v11 of discord.js ¯\_(ツ)_/¯

**Pull request merged**